insights/combiners/tmpfilesd.py

```diff
@@ -0,0 +1,74 @@
+"""
+Tmpfilesd configuration
+=======================
+
+The tmpfilesd files are normally available to rules as a list of TmpFilesd
+objects. This combiner turns those into one set of data, and provides a
+``find_file()`` method to search for that filename among all the files.
+"""
+
+from insights.core.plugins import combiner
+from insights.parsers.tmpfilesd import TmpFilesD
+
+
+@combiner(TmpFilesD)
+class AllTmpFiles(object):
+    """
+    Combiner for accessing all the tmpfilesd configuration files. Configuration
+    files can be found in three directories: ``/usr/lib/tmpfiles.d``,
+    ``/run/tmpfiles.d``, and ``/etc/tmpfiles.d``. Each directory overrides the
+    settings in the previous directory. This combiner checks for and accounts
+    for this behavior.
+
+    Attributes:
+        files(set): the set of files found in all data files.
+        rules(dict): a dictionary of rules using the config file as the key
+        file_paths(list): a list of the file paths for the configurations.
+    """
+    def __init__(self, tmpfiles):
+        rules = {}
+        files = set([])
+        self.tmpfiles = tmpfiles
```

Comment:
Since the `combiner` will parse all the data passed from each `parser`, better do not store the original data of the `parser` here.